config/locales/cy.yml

```diff
@@ -289,51 +289,23 @@ cy:
 
     result:
       adviser: mae gan gynghorydd
-      miles_away: o filltiroedd i ffwrdd
-      miles_away_alt: less than a mile away
-      office: Prif swyddfa
-
-      contact_details:
-        email: E-bost
-        url: Gwefan
-        address: Cyfeiriad
-        telephone_number: Ffôn
-        telephone_number_sr_text: Ffoniwch %{firm_name} ar
-
-      recieve_advice:
-        heading: Cael cyngor
-        items:
-          - text: Yn eich cartref
-          - text: Yn swyddfa’r cwmni
-          - text: Yn rhywle a gytunwyd
-          - text: Ar-lein
-          - text: Ffôn
-      type_of_advice:
-        heading: Math o gyngor
-        types:
-          retirement_income_products: 'Cronfa(feydd) / cynilion pensiwn'
-          pension_transfer: Trosglwyddo pensiwn
-          options_when_paying_for_care: Talu am ofal
-          equity_release: Rhyddhau ecwiti
-          inheritance_tax_planning: Cynllunio ar gyfer treth etifeddiant
-          wills_and_probate: Ewyllysiau a phrofiant
+      miles_away: "%{distance} o filltiroedd i ffwrdd"
+      firm_has: ""
```

Comment:
Sorry I should have mentioned that. It's intentionally blank. The Welsh translation of the English string was too complex and so we resorted to a different format in the Welsh.